[camera] Fix video recording exception on Android (#3375)
flutter/plugins: packages/camera/camera/CHANGELOG.md

## 0.6.3+2

> * Fixes stopping video recordings not working on Android before a picture has been taken.

Reviewer: nit: sentence is not running nicely, maybe update it to something like:
```suggestion
* Fixes crash on Android which occurs after video recording has stopped just before taking a picture.
```
